ReportPortal.ATM/FrameworkFacade/FrameworkFacade.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
```

Comment:
Let's change the implicit usings to disable.

Reply:
Changed to disable